### ISSUE TITLE
Frog-jump-buffer display does not display buffer names, only letters for selection (Emacs 27.1)

### ISSUE THREAD

**Issue by nausher** [waymondo/frog-jump-buffer#20]
I installed frog-jump-buffer and restarted emacs.
When I try to invoke the frog-jump-buffer the pop-up menu only displays the letters for selection, I cannot see the buffer names. 
<img width="623" alt="image" src="https://user-images.githubusercontent.com/79359/93942072-8c037000-fce4-11ea-8d2a-601ec53bcacc.png">

It does appear that the selection itself is working. As when I press one of the displayed letters I am taken to a buffer/file.

**Comment by waymondo** [comment]
What theme are you using? Is it possible your posframe face's foreground color matches its background color?

**Comment by nausher** [comment]
I had to change the background color of `frog-menu-posframe-background-face` to make the menu items visible.
Posting steps for anyone else who faces this issue.

I followed this answer on stackoverflow - https://stackoverflow.com/questions/57447830/how-to-set-color-of-flycheck-posframe.

**Steps to customize** 
1) Customize face - `M-x customize-face`
2) Type the background face -  `frog-menu-posframe-background-face`
3) Choose the color and "Apply & Save" to apply for all future sessions.
I chose Black to make the light text become visible.

<img width="630" alt="image" src="https://user-images.githubusercontent.com/79359/94982520-f63fc000-04ef-11eb-82c1-17a26a2d8c4e.png">
